src/klein/resource.py

```diff
@@ -280,3 +280,39 @@ def write_response(r):
         d.addCallback(write_response).addErrback(log.err, _why="Unhandled Error writing response")
 
         return server.NOT_DONE_YET
+
+
+
+class KleinHTTPRequest(server.Request):
+
+    def getArg(self, key):
+        """
+        Get a single arg value.
```

Comment:
You forgot to document `@param key`, in particular `@type key`, which seems like rather the whole point of this change :).